web/src/types.ts

```diff
@@ -30,12 +30,15 @@ interface StateZoneDataForTimePeriod {
 export interface StateZoneData {
   co2intensity: number; //TODO https://linear.app/electricitymaps/issue/ELE-1495/update-app-backend-variable-naming-to-use-camel-case-update-the
   co2intensityProduction: number;
+  estimationMethod: string;
+  estimatedPercentage: number;
   fossilFuelRatio: number;
   fossilFuelRatioProduction: number;
   renewableRatio: number;
   renewableRatioProduction: number;
   stateDatetime: number;
   zoneKey: string;
+  hasOutage: boolean;
```

Comment:
```suggestion
  hasOutage?: boolean;
```

We should not assume these are always available, some are only available on certain aggregations and hasOutage should only be added to the data if there is an outage, otherwise it will not be included.